src/components/fieldset/_fieldset.scss

```diff
@@ -20,12 +20,26 @@
     box-sizing: border-box; // 1
     display: table;         // 2
     max-width: 100%;        // 1
+    margin-bottom: $govuk-spacing-scale-4;
     padding: 0;
     // Hack to let legends or elements within legends have margins in webkit browsers
     overflow: hidden;
 
     color: $govuk-text-colour;
     white-space: normal;    // 1
+
+    .govuk-heading-s {
+      margin-bottom: 0;
+    }
+
+    .govuk-heading-m {
+      margin-bottom: $govuk-spacing-scale-1;
+    }
+
+    .govuk-heading-l,
+    .govuk-heading-xl {
+      margin-bottom: $govuk-spacing-scale-3;
+    }
   }
 
   .govuk-c-fieldset__legend--bold {
```

Comment:
Not strictly part of the PR but I noticed that `govuk-c-fieldset__legend--bold` is not used anywhere so it could be removed from here. To make text bold, passing `govuk-!-w-bold` works well.

Reply:
I think it's a useful variant to have, even if we don't have an example of it.